service/src/main/java/com/ge/predix/acs/zone/management/dao/AttributeConnectorEntity.java

```diff
@@ -0,0 +1,50 @@
+package com.ge.predix.acs.zone.management.dao;
+
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "attribute_connector")
+public class AttributeConnectorEntity {
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "refresh_interval_hours", nullable = true)
+    private int refreshIntervalHours;
```

Comment:
@sanjeevchopra discussed with me he wanted the field in the ACS API to be in hours so that the minimum time they can pass is 1 hour. Sanjeev, what do you think? We could:

1. Change customer field in the public API to be in seconds
2. Convert from hours to seconds when we store in the DB
3. Keep it the way it is